### ISSUE TITLE
Arrival and departure date shifts a day on booking review

### ISSUE THREAD

**Issue by TEUCG** [WoollyinWalesIT/jomres#399]
When customers pick a date for arrival and for departure, the date shifts to the day before the one they chose when reviewing their booking request.

**Customer setting their check-in and check-out dates:**
![check-in-out-dates-set](https://user-images.githubusercontent.com/2991240/55982876-afe90800-5c4e-11e9-9604-0f00661a8622.JPG)

**Customer reviewing their booking shortly after:**
![check-in-out-dates-shifted-1-day](https://user-images.githubusercontent.com/2991240/55983042-0d7d5480-5c4f-11e9-9147-c6ea4ed56185.JPG)

**Comment by jomres** [comment]
Is your CMS installation set to UTC? UTC is always recommended.

**Comment by TEUCG** [comment]
> Is your CMS installation set to UTC? UTC is always recommended.

I just set it back to UTC - which makes Jomres booking system appear to work again.
But it breaks most core Joomla components and modules that rely on the timezone date/time.

Thus you are not using date functionality in a way compatible with most other Joomla extensions, and certainly not the Joomla way.

I live in California so the attached image and description are the consequences.

My articles now saves by default date to UTC / greenwich time-zone as if I created an article 7 hours in the future. Also when I set something to publish / unpublish automatically in the future I have to constantly calculate a 7 hour offset or my module or article will publish / unpublish 7 hours earlier than intended.

![image](https://user-images.githubusercontent.com/2991240/56622330-7320f780-65e4-11e9-8835-954c4851b815.png)

The image show my current time in the time zone the server and all managers operate in which is set in my Operating System in the grey block, and the result in a Joomla article saved with default settings.

Also when someone inputs data in the input field the field should accept the date the user inputs, and not change after saving. No calculations should be performed after requesting a date, as the customer requests a date not a calculation of dates.
If rules are set for certain dates it should relate to the time at the timezone that the rental is occupying, not the time at UTC.

This is a bug and recommending I use UTC is a bad recommendation.
